app_flutter/lib/task_helper.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_service/protos.dart' show Task;
+
+/// A helper class for common utilities done with a [Task].
+class TaskHelper {
```

Comment:
See https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

Its okay for the strings, but I would suggest leaving the methods as top levels

Reply:
Added. Thanks for sharing the resource!

Reply:
I thought Jonah's comment is about moving the methods out of the class.

Reply:
`... you don’t need a class just to define something.` This is a cool feature!

I moved the helper functions to the top level, and left the strings in a StageName class. This should be easy to swap out when the enums are created in the protobuf